.github/dependabot.yml

```diff
@@ -1,16 +1,209 @@
 version: 2
 updates:
-  - package-ecosystem: "github-actions"
-    directory: "/"
```

Comment:
This is not enough to update local actions.
For example, we use `docker/build-push-action@v5` in `.github/actions/build_docker_image/action.yml` even though v6 is available